share/functions/__fish_is_zfs_feature_enabled.fish

```diff
@@ -1,17 +1,7 @@
-function __fish_is_zfs_feature_enabled -a feature target -d "Returns 0 if the given ZFS feature is available or enabled for the given full-path target (zpool or dataset), or any target if none given"
-    type -q zpool
-    or return
-    set -l pool (string replace -r '/.*' '' -- $target)
-    set -l feature_name ""
-    if test -z "$pool"
-        set feature_name (zpool get -H all 2>/dev/null | string match -r "\s$feature\s")
-    else
-        set feature_name (zpool get -H all $pool 2>/dev/null | string match -r "$pool\s$feature\s")
-    end
-    if test $status -ne 0 # No such feature
-        return 1
-    end
-    set -l state (echo $feature_name | cut -f3)
-    string match -qr '(active|enabled)' -- $state
-    return $status
+function __fish_is_zfs_feature_enabled \
+    -a feature pool \
+    -d "Returns 0 if the given ZFS pool feature is active or enabled for the given pool or for any pool if none specified"
+
+    type -q zpool && \
```

Comment:
Instead of doing this awkward `&& \`, just use `and` on the next line, or do `|| return` on this.

I'm pretty sure `fish_indent`, our autoformatter, would remove all your line continuations here.

Reply:
> Instead of doing this awkward `&& \`, just use `and` on the next line, or do `|| return` on this.

Fixed 👍🏻 

> I'm pretty sure `fish_indent`, our autoformatter, would remove all your line continuations here.

That's true, it'd would. Good point about the formatter, wasn't aware it exists. It fixed a single whitespace there.

Thanks for the quick response :) Great project, I use it everyday 💪🏻❤️